docs/posts/2023-06/article3.md

```
@@ -31,7 +31,7 @@ order: 4
 
 “真的整理好了，我发誓。”
 
-![](./res/illustration/LOFTER@左刀行（潮石镇1）.webp){style="float:right;max-width:40%;margin-left:1em"}
+![](./res/illustration/LOFTER@左刀行（潮石镇1）.webp){.image-right-float style="max-width: 40%;"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)